layouts/community/ortho_4x12/junonum/keymap.c

```diff
@@ -120,39 +123,38 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 };
 
 #ifdef AUDIO_ENABLE
-float plover_song[][2]    = SONG(PLOVER_SOUND);
-float plover_gb_song[][2] = SONG(PLOVER_GOODBYE_SOUND);
-
-float caps_song_on[][2]  = SONG(NUM_LOCK_ON_SOUND);
-float caps_song_off[][2] = SONG(SCROLL_LOCK_ON_SOUND);
-
-// Custom songs for DPAD layer status
-float dpad_song_on[][2] = {
-	HD_NOTE(_A4), HD_NOTE(_A4), HD_NOTE(_A4),
-	QD_NOTE(_F4), QD_NOTE(_C5), HD_NOTE(_A4),
-	QD_NOTE(_F4),  QD_NOTE(_C5), WD_NOTE(_A4)
-};
-
-float dpad_song_off[][2] = {
-	ED_NOTE(_C6), ED_NOTE(_C6),
-	ED_NOTE(_C6), ED_NOTE(_C6),
-	W__NOTE(_REST), QD_NOTE(_GS5),
-	QD_NOTE(_AS5), Q__NOTE(_C6),
-	Q__NOTE(_AS5), Q__NOTE(_C6)
-};
+    float plover_song[][2]    = SONG(PLOVER_SOUND);
```

Comment:
you may want to run `qmk format-c` on this file, as it should format everything for you

Reply:
It flattens the layers and de-indent all the preprocessor directives, which is the opposite of I'm trying to fix. Thanks for the suggestion regardless.

Reply:
For this particular line, I'm trying align the assigned value with the next line.